modules/aruco/include/opencv2/aruco/dictionary.hpp

```diff
@@ -0,0 +1,157 @@
+/*
+By downloading, copying, installing or using the software you agree to this
+license. If you do not agree to this license, do not download, install,
+copy or use the software.
+
+                          License Agreement
+               For Open Source Computer Vision Library
+                       (3-clause BSD License)
+
+Copyright (C) 2013, OpenCV Foundation, all rights reserved.
+Third party copyrights are property of their respective owners.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+  * Redistributions of source code must retain the above copyright notice,
+    this list of conditions and the following disclaimer.
+
+  * Redistributions in binary form must reproduce the above copyright notice,
+    this list of conditions and the following disclaimer in the documentation
+    and/or other materials provided with the distribution.
+
+  * Neither the names of the copyright holders nor the names of the contributors
+    may be used to endorse or promote products derived from this software
+    without specific prior written permission.
+
+This software is provided by the copyright holders and contributors "as is" and
+any express or implied warranties, including, but not limited to, the implied
+warranties of merchantability and fitness for a particular purpose are
+disclaimed. In no event shall copyright holders or contributors be liable for
+any direct, indirect, incidental, special, exemplary, or consequential damages
+(including, but not limited to, procurement of substitute goods or services;
+loss of use, data, or profits; or business interruption) however caused
+and on any theory of liability, whether in contract, strict liability,
+or tort (including negligence or otherwise) arising in any way out of
+the use of this software, even if advised of the possibility of such damage.
+*/
+
+#ifndef __OPENCV_DICTIONARY_HPP__
+#define __OPENCV_DICTIONARY_HPP__
+
+#include <opencv2/core.hpp>
+
+namespace cv {
+namespace aruco {
+
+//! @addtogroup aruco
+//! @{
+
+
+/**
+ * @brief Dictionary/Set of markers. It contains the inner codification
+ *
+ */
+class CV_EXPORTS Dictionary {
+
+    public:
+    Mat bytesList;         // marker code information
```

Comment:
As you have said, this is not mentioned in the OpenCV Coding Style. Thus I have followed the same convention I have seen in other parts of the library.
The convention is not full clear, but in general it depends on whether the member variable is ```public``` or ```private```.
Private member variables sometimes include an underscore at the beginning ( ```_variable``` ), while public member variables do not have any mark.

There are several samples here ( classes ```Mat```, ```UMat```, ```SparseMat```, ```RNG```, ```SVD```, etc ):
https://github.com/Itseez/opencv/blob/master/modules/core/include/opencv2/core/mat.hpp
https://github.com/Itseez/opencv/blob/master/modules/core/include/opencv2/core.hpp

In this case, since variables are public I have not put any mark. For instance, in the ```CharucoBoard``` class, private variables have an underscore at the beginning.

Many style guides encourage not to use public variables inside classes, so maybe a better approach would be setting these variables as ```private``` (with their corresponding underscores) and creating the corresponding getters and setters.